modules/mo_pytorch/mo_extensions/load/pytorch/hooks.py

```diff
@@ -158,24 +203,47 @@ def __init__(self, begin, end, begin_mask, end_mask, shrink_mask):
 
         return forward_hook(sslice, (self,), res)
 
+    # a * value
+    def __mul__(self, a):
+        if isinstance(a, OpenVINOTensor):
+            class Mul(nn.Module):
+                pass
+            res = self._value * a._value
+            return forward_hook(Mul(), (self, a), res)
+
+        elif isinstance(a, float):
```

Comment:
We also need to consider non-float values

Reply:
Done, thanks!